### PR TITLE
Integrating VFS, Linking GFS IOs, pining project version to 2024.16, …

### PR DESCRIPTION
**Pull request by janezg-SLAC**
…run pre-commit

Adding fast valve shutter to the code. To do that we also had to create new task which runs PRG_FSV. New task cycle time is set to 2ms and priority set to 15

Code was pushed to PLC and new configuration activated. 
The fast shutter valve can be open and closed remotely using pydm screen. To be able to open the fast shutter valve I had to force the trigger input on the PLC to false since the pressure is not good enough at the moment. Valve opened successfully and the status readback was correct. I released the forced values and fast shutter valve closed immediately. The valve behavior is as expected. 
GFS can be turned ON/OFF remotely using pydm screen. 

## Pre-merge checklist
- [x] Code works interactively
- [x] Code contains descriptive comments
- [x] Test suite passes locally
- [x] Libraries are set to fixed versions and not ``Always Newest``
- [x] Code committed with ``pre-commit`` (alternatively ``pre-commit run --all-files``)

### CODE REVIEW

File: plc-txi-hxr-vac/txi_hxr_vac/POUs/PRG_TXI_VAC_FS_Interfaces.TcPOU [hunk 1]
```diff
@@ -0,0 +1,13 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<TcPlcObject Version="1.1.0.1" ProductVersion="3.1.4024.6">
+  <POU Name="PRG_TXI_VAC_FS_Interfaces" Id="{a407d89f-23be-48cc-8c55-320fe1bc2eb0}" SpecialFunc="None">
+    <Declaration><![CDATA[PROGRAM PRG_TXI_VAC_FS_Interfaces
+VAR
+END_VAR
+]]></Declaration>
+    <Implementation>
+      <ST><![CDATA[//Fast Shutter Valve Interface
+TV2L1_VFS_1_Interface(IG := fb_TV4L1_PLEG_GFS_01.IG, Veto_valve := fb_TV2L1_PLEG_VGC_01.iq_stValve); //Veto Valve? Closest to fast shutter?]]></ST>
```
[the remark this answers is not in the document]
Comment (jyotiphy) [reply plc-txi-hxr-vac/txi_hxr_vac/POUs/PRG_TXI_VAC_FS_Interfaces.TcPOU:10]: @jyin999 Any better suggestion for Veto Valve here ?

File: plc-txi-hxr-vac/txi_hxr_vac/POUs/PRG_TXI_VAC_FS.TcPOU [hunk 2]
```diff
@@ -7,7 +7,7 @@ END_VAR
 ]]></Declaration>
     <Implementation>
       <ST><![CDATA[//Fast Shutter Valve
-fb_TV2L1_VFS_1(
+TV2L1_VFS_1(
```
[the remark this answers is not in the document]
Comment (jyotiphy) [reply plc-txi-hxr-vac/txi_hxr_vac/POUs/PRG_TXI_VAC_FS.TcPOU:10]: This seems fine.

**Review by jyotiphy — APPROVED** [review]
Looks fine to me! Just added Jing for review to see if she has suggestion for Veto Valve

File: plc-txi-hxr-vac/txi_hxr_vac/POUs/PRG_TXI_VAC_FS.TcPOU (hunk 2 above)
[the remark this answers is not in the document]
Comment (janezg-SLAC) [reply plc-txi-hxr-vac/txi_hxr_vac/POUs/PRG_TXI_VAC_FS.TcPOU:10]: image
Yes, you linked it here.

**Pull request merged**